packages/@react-spectrum/dialog/docs/DialogTrigger.mdx

```diff
@@ -0,0 +1,464 @@
+    <Heading>Version Info</Heading>
+    <Divider />
+    <Content>
+      <Text>
+        Version 1.0.0, Copyright 2020
+      </Text>
+    </Content>    
+  </Dialog>
+</DialogTrigger>
+```
+
+## Dialog Placement
+
+Popover Dialogs support a variety of placement options since they do not take over the user interface
+like Modal or Tray Dialogs.
+
+### Dialog Anchor
+
+By default, Popover Dialogs anchor themselves to their associated trigger. This can be overridden by providing
+a separate ref to the `targetRef` prop.
+
+```tsx example
+function Example() {
+  let ref = React.useRef()
+
+  return (
+    <div>
+      <DialogTrigger type="popover" targetRef={ref}>
+        <ActionButton>Trigger</ActionButton>
+        <Dialog>
+          <Heading>The Heading</Heading>
+          <Divider />
+          <Content>
+            <Text>
+              This is a popover anchored to the span.
+            </Text>
+          </Content>
+        </Dialog>
+      </DialogTrigger>
+      <span ref={ref} style={{marginInlineStart: '200px', width: '50px'}}>Popover appears over here</span>
+    </div>
+  );
+}
+```
+
+### Placement
+
+The Popover Dialog's placement with respect to its anchor element can be adjusted using the `placement`
+prop. See the props table [below](#props) for a full list of available placement combinations.
+
+```tsx example
+<DialogTrigger type="popover" placement="right top">
+  <ActionButton>Trigger</ActionButton>
+  <Dialog>
+    <Heading>The Heading</Heading>
+    <Divider />
+    <Content>
+      <Text>
+        This is a popover placed to the right of its trigger and offset so the arrow is at the top of the dialog.
+      </Text>
+    </Content>
+  </Dialog>
+</DialogTrigger>
+```
+
+### Offset and Cross Offset
+
+The Popover Dialog's offset with respect to its anchor element can be adjusted using the `offset` and
+`crossOffset` props. `offset` controls the spacing between the Dialog and the anchor element parallel 
+to the direction the Dialog is rendered whereas `crossOffset` handles the offset in the direction perpendicular. 
+
+```tsx example
+<DialogTrigger type="popover" placement="top" offset={-50}>
+  <ActionButton>Trigger</ActionButton>
+  <Dialog>
+    <Heading>The Heading</Heading>
+    <Divider />
+    <Content>
+      <Text>
+        This is a popover offset by an additional 50px above the trigger.
+      </Text>
+    </Content>
+  </Dialog>
+</DialogTrigger>
+
+<DialogTrigger type="popover" placement="top" crossOffset={100}>
+  <ActionButton>Trigger</ActionButton>
+  <Dialog>
+    <Heading>The Heading</Heading>
+    <Divider />
+    <Content>
+      <Text>
+        This is a popover offset by an additional 100px to the right of the trigger.
+      </Text>   
+    </Content>
+  </Dialog>
+</DialogTrigger>
+```
+
+### Flipping
+
+Applying `shouldFlip` to the DialogTrigger makes the Popover Dialog attempt to flip
+on its main axis in situations where the original placement would cause it to render out of view.
```

Comment:
Should we have an example button that doesn't flip? Flipping seems to be the default so it would be nice to see not flipping.

Reply:
I can add an example below